pydicom/tests/conftest.py

```diff
@@ -3,24 +3,31 @@
 """Fixtures used in different tests."""
 
 import pytest
-
 from pydicom import config
 
 
 @pytest.fixture
 def enforce_valid_values():
-    value = config.enforce_valid_values
-    config.enforce_valid_values = True
```

Comment:
Maybe we have to keep this tests in addition to the new ones - based on my other comment, to check that old code still behaves the same?

Reply:
I think we should add a separate test for this instead of using it for the fixture.

Reply:
Done.